### ISSUE TITLE
Unable to get it working with version 4

### ISSUE THREAD

**Issue by illank86** [larsonnn/strapio#10]
Hi,

i have followed the readme to get strapio working with v4 but, i got 404 notFound..

i have tried to harcoded the JWT token, but still unable to get the socket.io working.

**Comment by x0ptr** [comment]
which strapio version do you use?

**Comment by x0ptr** [comment]
forgot to update the README for the bootstrap.js.

You need to write the bootstrap logic in  `src/index.js`

```
bootstrap({strapi}) {
        process.nextTick(() => {
                strapi.StrapIO = (require("strapio"))(strapi);
        });
 },
 
 ```
 
 I update the README as soon im at home. :)

**Comment by x0ptr** [comment]
closed with 4d04a51409f4115ad631366167479d2f98198e1b
